tbxforms/forms.py

```diff
@@ -31,6 +33,16 @@ def helper(self) -> FormHelper:
         fh.legend_size = Size.MEDIUM
         return fh
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # Escape HTML in `label` and `help_text` unless it's set to allow.
+        for field_name, field in self.fields.items():
+            if not getattr(settings, "TBXFORMS_ALLOW_HTML_LABEL", False):
+                field.label = conditional_escape(field.label)
+            if not getattr(settings, "TBXFORMS_ALLOW_HTML_HELP_TEXT", False):
+                field.help_text = conditional_escape(field.help_text)
```

Comment:
@kbayliss could you add unit tests for this as well? Feels like something we definitely want to make sure there will never be regressions on.

Reply:
Good shout, I was rushing to get this done and merged. I've opened https://github.com/torchbox/tbxforms/issues/20 for this.